src/NuGet.Services.Staging.Common/NuGet.Services.Logging.xproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
```

Comment:
Forgot to remove it.

---
In reply to: [74158058](https://github.com/NuGet/NuGet.Staging/pull/69#discussion_r74158058) [](ancestors = 74158058)